db/large_data_handler.cc

```diff
@@ -146,9 +146,9 @@ future<> cql_table_large_data_handler::record_large_cells(const sstables::sstabl
     if (clustering_key) {
         const schema &s = *sst.get_schema();
         auto ck_str = key_to_str(*clustering_key, s);
-        return try_record("cell", sst, partition_key, int64_t(cell_size), cell_type, format("{} {}", ck_str, column_name), extra_fields, ck_str, column_name);
+        return try_record("cell", sst, partition_key, int64_t(cell_size), cell_type, format("/{}/{}", ck_str, column_name), extra_fields, ck_str, column_name);
```

Comment:
I'm afraid I don't understand this change or why try_record even takes this parameter which you changed here...
According to docs/dev/system_keyspace.md, the partition key, clustering key, and column name, are all stored individually in the table. This is also why this function try_record() takes partition_key, ck_str, and column_name separately. So why do we ned to pass it, additionally, this concatenated format? why can't it calculate it on its own?

Reply:
Also, you forgot to mention in the commit message (or it wasn't clear enough to me) that what you're changing here is just the log messages - NOT the large-cell/row system table - which is why we think it's ok to change the format - we are not backward compatible when it comes to log messages.

Reply:
Right.  This changes only the log message, that need not be backward compatible (it was buggy anyhow by concatenating the partition and clustering keys together, with no delimiter).
We can also consider printing it in a more structured way like, `pk=<partition_key> ck=<clustering_key> column=<column_name>` instead of `<partition_key>/<clustering_key>/<column_name>` that might be useful for developers that may use the code to parse the log message, but less useful to users - who should consult the system.large_* tables rather than the log.